final-project/manConstraintsJustification.tex

```diff
@@ -25,3 +25,11 @@ \subsection{Issues with Stateless Balancer}
 \section{Balancer Must Use Asynchronous I/O \index{async} \index{asynchronous I/O} \index{io}}
 
 The Balancer's workload is I/O bound, and it must be able to handle many concurrent network connections. For I/O bound workloads, it's more performant to use asynchronous I/O\cite{async-vs-threads}.
+
+\section{Current Deployments Must Continue to Work}
+
+Once the load balancer is implemented, the current deployments of OTT must continue to function as intended. Likewise, any updates to the Monolith should not affect the functionality of current deployments.
+
+\section{Deployment Must Work Without Load Balancer}
+
+It must remain possible to deploy OTT without the load balancer. 
```

Comment:
The load balancer must remain optional because self-hosters might not want to add the additional complexity to their deployments. Also, it would be beneficial to be able to quickly reroute traffic between the balancer and the monolith during initial deployments where we're still working out kinks.

You might want to reword that a little so it fits into the paper better.